spec/rspec/mocks/mutate_const_spec.rb

```diff
@@ -218,6 +218,12 @@ def change_const_value_to(value)
           it_behaves_like "loaded constant stubbing", "TestClassThatDefinesSend::C"
         end
 
+        context "for a non string argument" do
+          it { expect { stub_const(10, 1) }.to raise_error(ArgumentError, /Please provide a string/i) }
+          it { expect { stub_const(nil, 1) }.to raise_error(ArgumentError, /Please provide a string/i) }
+          it { expect { stub_const(10.1, 1) }.to raise_error(ArgumentError, /Please provide a string/i) }
+        end
```

Comment:
I don't think we need 3 specs for this--one is sufficient. And a descriptive doc string would be nice.  So, instead of a context, how about:

``` ruby
it "requires a string argument" do
  expect { ... }.to raise_error(...)
end
```